numba_dpex/tests/experimental/test_strided_dpnp_array_in_kernel.py

```diff
@@ -36,74 +103,192 @@ def change_values_3d(x, v):
     x[i, j, k] = p  # setitem
 
 
-def test_strided_dpnp_array_in_kernel():
+def change_values_3d_func(a, p):
+    """Assign values in a 3d numpy.ndarray
+
+    Args:
+        a (numpy.ndarray): Input array.
+        p (int): Value to be assigned.
+    """
+    for i in range(a.shape[0]):
+        for j in range(a.shape[1]):
+            for k in range(a.shape[2]):
+                a[i, j, k] = p
+
+
+@pytest.mark.parametrize("N", [8, 16, 32, 64, 128, 256, 512])
```

Comment:
Why do we need so many `N` values? I do not feel they add much to the test coverage and only make the suite run longer.

Reply:
fixed.